sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -813,3 +813,91 @@ export type RequireAtLeastOne<T> = {
  * which are: emailAddress, name or phone.
  */
 export type CertificateContact = RequireAtLeastOne<CertificateContactAll> | undefined;
+
+/** Known values of {@link CertificateKeyCurveName} that the service accepts. */
```

Comment:
These enums used to be re-exported from the generated code, but they no longer have doc-comments. Instead of reverting the changes in generated layer and fighting with this every time I regenerate I just copied them to convenience layer instead

Reply:
yeah generally we don’t like re-exporting from the generated code, because it tends to change. So I think this is better anyway